docs/program/apis/cloud.md

```diff
@@ -0,0 +1,206 @@
+---
+title: "Managing Your Fleet with Viam's Cloud API"
+linkTitle: "Cloud Management"
+weight: 20
+type: "docs"
+description: "How to use the cloud API with Viam's client SDKs."
+tags: ["cloud", "sdk", "viam-server", "networking", "apis", "robot api", "cloud management"]
+---
+
+The cloud app API allows you to [manage your robot fleet](/manage/fleet/) with code instead of through the [Viam app](https://app.viam.com/).
+Functionality includes:
+
+- create and manage organizations, locations, and individual robots
+- manage permissions and authorization
+- create and manage fragments
+
+{{% alert title="Support Notice" color="note" %}}
+
+As of August 24, 2023, cloud app API methods are only available in the Python SDK.
+
+{{% /alert %}}
+
+## API
+
+{{% alert title="Tip" color="tip" %}}
+
+To use the Viam cloud app API, you need to instantiate a [`ViamClient`](https://python.viam.dev/autoapi/viam/app/viam_client/index.html#viam.app.viam_client.ViamClient) and then instantiate an [`AppClient`](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient):
+
+```python {class="line-numbers linkable-line-numbers"}
+from viam.rpc.dial import DialOptions, Credentials
+from viam.app.viam_client import ViamClient
+
+async def connect() -> ViamClient:
+    dial_options = DialOptions(
+        auth_entity='mrroboto.this_is_just_an_example.viam.cloud',  # The URL of your robot.
+        credentials=Credentials(
+            type='robot-location-secret',
+            payload='YOUR ROBOT LOCATION SECRET' # The location secret of your robot
+        )
+    )
+    return await ViamClient.create_from_dial_options(dial_options)
+
+async def main():
+
+  # Make a ViamClient
+  viam_client = await connect()
+  # Instantiate an AppClient called "cloud" to run the cloud app API methods on
+  cloud = await viam_client.app_client()
+```
+
+Then, run methods on the `AppClient` according to the code examples for each method below.
+
+{{% /alert %}}
+
+The cloud API supports the following methods:
+
+{{< readfile "/static/include/services/apis/cloud.md" >}}
+
+### ListOrganizations
+
+List the organizations the user is an authorized user of.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- None
+
+**Returns:**
+
+- [(List[viam.proto.app.Organization])](https://python.viam.dev/autoapi/viam/proto/app/index.html#viam.proto.app.Organization): A list of the organization or organizations of which the user is an authorized owner.
+
+```python {class="line-numbers linkable-line-numbers"}
+org_list = await cloud.list_organizations()
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient.list_organizations).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### ListOrganizationMembers
+
+List the members and invites of the currently authenticated-to organization.
```

Comment:
```suggestion
List the members and invites of the organization that you are currently authenticated to.
```
Speak directive to the user-- sentence is bulkier than it needs to be

Reply:
Yeah--I was just using the verbiage from the Python SDK docs but you're right this is better. Changing.